src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
```

Comment:
Add an empty line between multiline sibling blocks of HTML to add some 'air' and simplify reading. However, make sure not to add them between parent and child elements. Refer to the checklist item 'Add empty lines between multiline sibling blocks of HTML'.